crates/codegen/runtime/cargo/src/runtime/generated/language.rs

```diff
@@ -62,6 +62,10 @@ impl Language {
         }
     }
 
+    pub fn root_kind() -> NonterminalKind {
```

Comment:
This is being added to the public API. is this expected?
When is this usable by users?

Reply:
Additionally, since it is never changed, I wonder if it can be a `const` instead of a `fn`, for better visibility/DX.

Reply:
I'm using this as the root kind for the implementation of the CLI parser. And since it's already part of the language definition, I thought it wouldn't hurt to expose it.

We can change it to a `const`, although for the stubs it's unclear what value would it take (I guess we could use one of the `StubX` placeholder variants).

Reply:
Sounds good.

Reply:
I changed this to be a const `ROOT_KIND` and is now also exported to JS (via a static method, as I didn't find a way to expose a Rust `const` inside a `struct`).